### DISCUSSION TITLE
stuck at Could not determine IPv4/IPv6 protocol

### DISCUSSION THREAD

**Discussion by usma0118** [haugene/docker-transmission-openvpn#2163] [Q&A] [answered]
Fri Jan 21 18:32:03 2022 Could not determine IPv4/IPv6 protocol
Fri Jan 21 18:32:03 2022 SIGUSR1[soft,init_instance] received, process restarting
Fri Jan 21 18:37:03 2022 NOTE: the current --script-security setting may allow this configuration to call user-defined scripts
Fri Jan 21 18:37:11 2022 RESOLVE: Cannot resolve host address: {redacted} (Temporary failure in name resolution)
Fri Jan 21 18:37:19 2022 RESOLVE: Cannot resolve host address: {redacted} (Temporary failure in name resolution)
Fri Jan 21 18:37:19 2022 Could not determine IPv4/IPv6 protocol
Fri Jan 21 18:37:19 2022 SIGUSR1[soft,init_instance] received, process restarting
Fri Jan 21 18:42:19 2022 NOTE: the current --script-security setting may allow this configuration to call user-defined scripts
Fri Jan 21 18:42:27 2022 RESOLVE: Cannot resolve host address: {redacted} (Temporary failure in name resolution)
Fri Jan 21 18:42:35 2022 RESOLVE: Cannot resolve host address: {redacted} (Temporary failure in name resolution)
Fri Jan 21 18:42:35 2022 Could not determine IPv4/IPv6 protocol
Fri Jan 21 18:42:35 2022 SIGUSR1[soft,init_instance] received, process restarting
Fri Jan 21 18:47:35 2022 NOTE: the current --script-security setting may allow this configuration to call user-defined scripts
Fri Jan 21 18:47:43 2022 RESOLVE: Cannot resolve host address: {redacted} (Temporary failure in name resolution)
Fri Jan 21 18:47:51 2022 RESOLVE: Cannot resolve host address: {redacted} (Temporary failure in name resolution)
Fri Jan 21 18:47:51 2022 Could not determine IPv4/IPv6 protocol
Fri Jan 21 18:47:51 2022 SIGUSR1[soft,init_instance] received, process restarting

**Comment by usma0118** [comment, marked as answer]
little too quick to post :).. DNS issue on my network. closing thread.

**Reply by adamh98** [reply]
@usma0118  What was the DNS issue on your network? I have the exact same error, but the container is able to reach PIA and google dns... it with nslookup.
